docs/content/en/docs/tutorials/tetragon-metrics.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Tetragon Metrics"
+weight: 1
+icon: "overview"
+description: "Fetching and understanding Tetragon metrics"
+---
+
+# Fetching Tetragon Metrics
+
+## Kubernetes
+
+Tetragon, when installed with Helm as mentioned in [Deploying on Kubernetes] ({{< ref "docs/getting-started/deployment/kubernetes" >}}), creates a pod with `tetragon-` prefix. It also creates a service named `tetragon` that exposes metrics on the specified port.
+
+### Getting metrics port
+
+Check if `tetragon` service is running
+
+```bash
+➜  ~ kubectl get services -n kube-system
+NAME       TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)                  AGE
+kube-dns   ClusterIP   10.96.0.10     <none>        53/UDP,53/TCP,9153/TCP   3m
+tetragon   ClusterIP   10.96.54.218   <none>        2112/TCP                 3m
+```
+
+Describe the tetragon service to see the port on which metrics are being exposed.
```

Comment:
```suggestion
Describe the `tetragon` service to see the port on which metrics are being exposed. The default port is `2112`.
```
Maybe let's document the default here?

Reply:
Makes sense, added that.

Reply:
@mtardy suggested removal of that part, since the output of 

```shell-session
kubectl get services tetragon -n kube-system
``` 

```
NAME       TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)     AGE
tetragon   ClusterIP   10.96.54.218   <none>        2112/TCP    3m
```

already displays the port. I added a note as suggestion.